libcnb-common/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+#![warn(clippy::pedantic)]
```

Comment:
(Sorry I missed this before) Should we add the `#![doc = include_str!("../README.md")]` that we use for all the other crates?